django/http/response.py

```diff
@@ -402,6 +402,11 @@ def __init__(self, streaming_content=(), *args, **kwargs):
         # See the `streaming_content` property methods.
         self.streaming_content = streaming_content
 
+    def __repr__(self):
+        class_name = self.__class__.__name__
+        status_code = self.status_code
+        return f'<{class_name} status_code={status_code}>'
```

Comment:
I would use the same format as in `HttpResponse.__repr__()`:
```suggestion
        return '<%(cls)s status_code=%(status_code)d%(content_type)s>' % {
            'cls': self.__class__.__qualname__,
            'status_code': self.status_code,
            'content_type': self._content_type_for_repr,
        }
```